src/game/game/game_demo.c

```diff
@@ -40,6 +45,14 @@ static void Game_Demo_LoadLaraPos(void)
     FLOOR_INFO *floor =
         Room_GetFloor(item->pos.x, item->pos.y, item->pos.z, &room_num);
     item->floor = Room_GetHeight(floor, item->pos.x, item->pos.y, item->pos.z);
+
+    Random_SeedDraw(0xD371F947);
+    Random_SeedControl(0xD371F947);
+
+    // LaraGun() expects request_gun_type to be set only when it
+    // really is needed, not at all times.
+    // https://github.com/LostArtefacts/TR1X/issues/36
```

Comment:
I think this comment about a super old, closed issue can be deleted? I don't think it's still relevant.

Reply:
It still is relevant:
1. Run normal game
2. Equip weapons
3. Use the pause screen (or likely the inventory screen) to exit to the main menu
4. Wait for the demo to run
5. Lara equips her pistols at the start of the sequence